test/e2e/pkg/client/oc.go

```diff
@@ -67,3 +71,16 @@ func (w *K8sClient) GetLogsForContainer(podName string, namespace, containerName
 	outBytes, err := cmd.CombinedOutput()
 	return string(outBytes), err
 }
+
+// function returns an empty string as `commandResult` to match the existing signature.
+func (w *K8sClient) OcDeleteWorkspace(name, namespace string) (commandResult string, err error) {
```

Comment:
nit, now that we're not relying on `oc`  we should move this method to `devws.go`

Reply:
done!